cli/Application/Command/Install/Install.php

```diff
@@ -68,7 +68,7 @@ public function execute()
 			if (!$this->getApplication()->input->get('reinstall'))
 			{
 				$this->out()
-					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found !!') . '</fg=black;bg=yellow>')
+					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found!') . '</fg=black;bg=yellow>')
 					->out()
 					->out(g11n3t('Do you want to reinstall ?'))
```

Comment:
While you are making these changes it would be good to fix this at the same time
There should be no space before the ?